bentoml/server/__init__.py

```diff
@@ -66,7 +66,7 @@ def start_dev_server(saved_bundle_path: str, port: int, enable_microbatch: bool)
         marshal_server.async_start(port=port)
         api_server.start()
     else:
-        api_server = BentoAPIServer(bento_service, port=port)
+        api_server = BentoAPIServer(bento_service, port=port, run_with_ngrok=run_with_ngrok)
```

Comment:
We would also want to apply this to `enable_microbatch=True` case above.

I'm more inclined to keep ngrok related logic here within the `start_dev_server` method, and move it to the beginning of this method:

```
        if run_with_ngrok:
            from flask_ngrok import start_ngrok
            thread = Timer(1, start_ngrok, args=(port,))
            thread.setDaemon(True)
            thread.start()

          if enable_microbatch:
             ...
             api_server.start()
          else:
             ...
            api_server.start()


```

Reply:
I agree with you. I'll make the required changes